src/plugins/hide-curriculum-strings.ts

```diff
@@ -38,7 +38,7 @@ const createChallengeTitleLookup = (
       },
     };
   } catch (err) {
-    console.log(JSON.stringify(err), null, 2);
+    console.log(JSON.stringify(err, null, 2));
```

Comment:
This one is quite funny...

Reply:
It took me far too long to figure out why `null 2` was printing.